packages/common/components/navbars/site-a.marko

```diff
@@ -19,35 +19,41 @@ $ const siteName = config.siteName();
           <img class=bem(navbar, 'logo') alt=siteName src=site.get('logos.navbar.src') srcset=site.getAsArray('logos.navbar.srcset').join(',')>
         </a>
         <ul class="site-navbar-items site-navbar-items--secondary">
-          <for|item| of=getNavItems({ site, regEnabled, hasUser, key: 'secondaryNavItems' })>
-            <li class=bem(nav, 'item')>
-              <endeavor-link class=bem(nav, 'link') target=item.target title=item.title href=item.href>${item.label}</endeavor-link>
-            </li>
-          </for>
+          <if(navigation.secondary && navigation.secondary.items)>
+            <for|item| of=getNavItems({ regEnabled, hasUser, items: navigation.secondary.items })>
```

Comment:
`<for|item| of=getNavItems({ regEnabled, hasUser, items: navigation.secondary })>`